src/content/en/2022/markup.md

```diff
@@ -1,22 +1,564 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Markup
-#TODO - Review and update chapter description
-description: Markup chapter of the 2022 Web Almanac covering the use of elements, attributes, document trends, as well as adoption rates of new standards.
-authors: [AlexLakatos, ibnesayeed, daKmoR]
-reviewers: [j9t, bkardell]
+description: Markup chapter of the 2022 Web Almanac covering document data (doctypes, compression, languages, HTML coformance, document size), the use of HTML elements and attributes, data attributes and social media.
+authors: [j9t]
+reviewers: [tunetheweb, bkardell]
 analysts: [rviscomi]
```

Comment:
@j9t can you confirm these contributors? I remove the others that I don't believe contributed but let me know if I've wrong.

I'm not sure if anyone other than myself reviewed your doc?
@bkardell you were down as a reviewer, but not sure if you did review or not? Know you're busy with your own chapter, but if you do want to review you can see the doc here: https://20220923t160735-dot-webalmanac.uk.r.appspot.com/en/2022/markup

Reply:
@zcorpan also [gave it a look](https://github.com/HTTPArchive/almanac.httparchive.org/issues/2881#issuecomment-1238156094) and should get reviewer credit. Could also be good to give it another look in the website preview and leave any additional feedback.

Reply:
Ah missed that. Good so @zcorpan and @bkardell reviewed. I'll remove my name and settle for editor. Just didn't want to give the impression it was not given the appropriate scrutiny, but seems it was!